tests/integration/cli/test_cli.py

```diff
@@ -119,6 +119,7 @@ def test_otx_e2e(recipe: str, tmp_path: Path) -> None:
         f"base.data_dir={DATASET[task]['data_dir']}",
         f"base.work_dir={tmp_path_test}",
         f"base.output_dir={tmp_path_test / 'outputs'}",
+        "trainer=gpu",
```

Comment:
It is not desired to restrict these tests only for a GPU device. Would you please reuse and expand this parameterization
https://github.com/openvinotoolkit/training_extensions/blob/e563354601a734c767623091c30c80975a8635de/tests/regression/conftest.py#L158-L160
with CPU?

You should also add a marker here
https://github.com/openvinotoolkit/training_extensions/blob/e563354601a734c767623091c30c80975a8635de/pyproject.toml#L300-L302

Reply:
Done by [db72f6c](https://github.com/openvinotoolkit/training_extensions/pull/2789/commits/db72f6c4abd756ec13503e41e1b424528510d143) Could you check it?